var/spack/repos/builtin/packages/sst-elements/package.py

```diff
@@ -159,6 +163,9 @@ def configure_args(self):
         if "+otf" in spec:
             args.append("--with-otf=%s" % spec["otf"].prefix)
 
+        if "+ariel_mpi" in spec:
+            args.append("--enable-ariel-mpi")
```

Comment:
(Optional follow-up) If possible, supplying one of the enable/disable or with/without flags results in behavior that avoids automatic finding of system dependencies in some cases.
```suggestion
            args.append(self.enable_or_disable("ariel-mpi"))
```
(In this particular case, care must be taken to add the `variant="ariel_mpi"` argument too, due to difference in dash/underscore.)